web/src/containers/ActivitiesList.js

```diff
@@ -30,33 +18,33 @@ const activities = [
 ];
 
 const ActivitiesList = ({ goTo }) => (
-  <Box py={4}>
-    <Heading mb={3}>Activities</Heading>
-    <Box mb={5}>
+  <div>
+    <h1>Activities</h1>
+    <div className="mb3">
       {activities.map(([name, status, href]) => (
-        <Border key={name} py={2} bottom>
-          <Relative>
-            <Absolute right>
-              {href === '#!' ? (
-                <Text color="gray">{status}</Text>
-              ) : (
-                <Link to={href} is={RRLink}>
-                  {status}
-                </Link>
-              )}
-            </Absolute>
-            {name}
-          </Relative>
-        </Border>
+        <div key={name} className="py1 relative border-bottom border-silver">
+          <div className="absolute right-0">
+            {href === '#!' ? (
+              <div className="gray">{status}</div>
+            ) : (
+              <Link to={href}>{status}</Link>
+            )}
+          </div>
+          {name}
+        </div>
       ))}
-    </Box>
-    <ButtonOutline mr={2} onClick={() => goTo('/activities-start')}>
+    </div>
+    <button
```

Comment:
For the sake of maintainability, should we consider creating a set of wrappers around these low-level components that we style the same way in various parts of the app?  We might end up with something like...

```js
import { BlueOutlineButton } from '../buttons'; // ignore the name, it's just to sketch the idea

...

<BlueOutlineButton onClick={}>Back</BlueOutlineButton>
```

I'm just thinking about the effort involved if we decide to change how those buttons look.  We'd have to change it in several places right now, which is both extra time and extra risk that we'll forget some.